calico_kubernetes/calico_kubernetes.py

```diff
@@ -1,23 +1,27 @@
 #!/bin/python
-import json
+from __future__ import print_function
```

Comment:
Right - I'd suggest adding a comment to explain that then, so someone doesn't come along and remove it without having that knowledge.